packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -116,7 +117,9 @@ export function initWrapperState(element: Element, props: Object) {
   }
 
   node._wrapperState = {
-    initialValue: '' + initialValue,
+    initialValue: getSafeValue(
+      props.value != null ? props.value : initialValue,
```

Comment:
This seems to be different than the previous implementation. Do you mind explaining why? 🙂

Reply:
Ah, you're right. This has to be a mistake. I took influence from `ReactDOMFiberInput` and the files got oddly similar under cognitive load.

Instead, it should can simply be `getSafeValue(initialValue)`, I think :-)

Thanks!

Reply:
`initialValue` already references `props.value` on L:83 so this logic is redundant.